specs/dataclass_transforms.txt

```diff
@@ -0,0 +1,399 @@
+Motivation
```

Comment:
This file appears to be markdown; can you make in an `md` so github formats it nicely?

Reply:
I've sent a file link to several people already. If I rename the file, those links will be invalidated.